### ISSUE TITLE
About the big gap between my evaluation results and the metrics (Localization AUC of pre-trained models) reported in the paper

### ISSUE THREAD

**Issue by ghost** [proteus1991/PSCC-Net#3]
Thank you very much for open-sourcing the relevant code, datasets, and pre-trained models. 

I used your open-source pre-trained model to infer directly on the public datasets including CASIAv1, COVERAGE, NIST16, IMD20, Columbia, and there is a big gap between the evaluation results and the metrics reported in the paper, and I also tried to retrain the dataset you provided, but generalization is hard to guarantee. Hope you can check the uploaded pretrained model, or provide the code for metric evaluation.
![image](https://user-images.githubusercontent.com/45398334/196845476-c8c9cf2a-6a5d-4904-9e0c-3b70ebac4531.png)

**Comment by chenxinshuang** [comment]
I used pretrained-model to evaluate these five public datasets,and there is a big gap between the evaluation and the result reported in the paper,too

**Comment by asdjia** [comment]
Hi, may I ask how did you download the NIST16 dataset with 564 images. I downloaded it, but the number of images is not the same(like about 1,000 images in total).

**Comment by chengyan23** [comment]
Hi,I also meet this problem,how do you solve this problem?
